stable/joomla/values.yaml

```diff
@@ -72,42 +72,42 @@ externalDatabase:
 mariadb:
```

Comment:
Can you reflect these changes in the README table?

https://github.com/kubernetes/charts/tree/master/stable/joomla#configuration

Reply:
done